src/styles/bank_transactions.scss

```diff
@@ -296,14 +296,23 @@
   display: flex;
   flex-direction: column;
   gap: var(--spacing-sm);
-  max-width: 680px;
 }
 
 .Layer__expanded-bank-transaction-row__content-panels {
   box-sizing: border-box;
   display: flex;
   flex-direction: row;
-  max-width: 680px;
+}
+
+.Layer__bank-transaction-form-fields {
+
+  .Layer__ComboBoxContainer {
+    max-width: 20rem;
+  }
+
+  .Layer__bank-transaction-memo-input-group {
+    max-width: 40rem;
+  }
```

Comment:
Acknowledging I'm stacking mud on mud here. Trying to do as little as possible to accommodate changes here without trying to bite off a full refactor of layout